resources/deu.ini

```diff
@@ -1,213 +1,207 @@
 # German language pack for MyPlot
-# Created by SalmonDE
+# Created by SalmonDE and morpheus132
```

Comment:
I'm just saying this if you would like credit if the project were ever removed from github